src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -22,6 +22,11 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<FunctionInvocationMiddleware>();
             builder.UseMiddleware<HostWarmupMiddleware>();
 
+            builder.UseWhen(context => !context.Request.Path.StartsWithSegments("/admin"), config =>
```

Comment:
Doesn't this regress us back away from the opt-in attribute model I put in place?

Reply:
No. The attribute still plays a role in actions that require the host to be running. No admin API will have the host check middleware, so the attribute is important.

Reply:
Unless you're referring to APIs outside of `admin`, which I don't think we have any plans to add, correct?

Reply:
Yes I am talking about new APIs - it would be good to have the design continue to be future proof.

Reply:
If we ever have additional APIs under a different resource/segment (something other than `admin`), we'd need to add them to this list here. We could make this smarter by checking the routes registered by the controllers, which would eliminate the need to update this, I just thought this was overkill for now.